frontend/src/app/components/MainInstallButton.js

```diff
@@ -31,13 +35,16 @@ export default class MainInstallButton extends React.Component {
   }
 
   renderInstallButton(isInstalling, hasAddon) {
+    let { experimentTitle } = this.props;
     return (
       <div>
         <button onClick={e => this.install(e)}
                 className={classnames('button extra-large primary install', { 'state-change': isInstalling })}>
           {hasAddon && <span className="progress-btn-msg" data-l10n-id="landingInstalledButton">Installed</span>}
           {!hasAddon && !isInstalling &&
-            <span className="default-btn-msg" data-l10n-id="landingInstallButton">Install the Test Pilot Add-on</span>}
+            <span>
+              <span className="default-btn-msg"><span data-l10n-id="landingInstallButton">Install the Test Pilot Add-on</span> { experimentTitle && "+" } { experimentTitle }</span>
```

Comment:
I wish I had some background on this PR (don't see an issue linked), but this is really bad. 

You should not hard-code pieces of strings (+), nor assume that all languages will have the same structure as English, and produce meaningful results from this kind of concatenation.